doc: Added working with PMICs section (#11311)
nrfconnect/sdk-nrf: doc/nrf/working_with_pmic/npm1300/features.rst

.. _ug_npm1300_features:

Features of nPM1300
###################

.. contents::
    :local:
    :depth: 2

Introduction
************

nPM1300 is a Power Management IC (PMIC) with a linear-mode lithium-ion, lithium-polymer, and lithium ferro-phosphate battery charger.

> It has two dual-mode buck regulators, two dual purpose LDO/load switches, 3 LED drivers and 5 GPIOs.

Reviewer: ```suggestion
It has two dual-mode buck regulators, two dual purpose LDO/load switches, three LED drivers and five GPIOs.
```

> nPM1300 is the perfect companion for nRF52, nRF53 series SoCs, and the nRF91 series SiP.

Reviewer: ```suggestion
nPM1300 is the perfect companion for nRF52 and nRF53 Series SoCs, and the nRF91 series SiP.
```

> It is ideal for compact and advanced IoT products such as advanced wearables and portable medical applications.

Reviewer: ```suggestion
It is ideal for compact and advanced IoT products such as wearables and portable medical applications.
```
